### ISSUE TITLE
Partial Shipment

### ISSUE THREAD

**Issue by shweta1234567** [magento/devdocs#1772]
Hello,

Can someone tell me that how can I create a partial shipment using Magento rest api.please review my JSON which I used for creating partial shipment?

{

	"order_item_id":
	[
		{
			
			"order_item_id":18486,
			"order_item_ids": 5
			
		},
		
		{
			
			"order_item_id":18487,
			"order_item_ids": 1
			
		},
		{
			
			"order_item_id":18488,
			"order_item_ids": 1
			
		}
		],
		"notify":true,
		"appendComment":false,
		"tracks":
		[
			{
				"track_number":"012356",
				"title":"",
				"carrier_code":"fedex"
				
			}
			]
	
}
but its creates full shipment in magento.

**Comment by keharper** [comment]
@shweta1234567 Have you checked http://devdocs.magento.com/guides/v2.2/get-started/order-tutorial/order-create-shipment.html ?

**Comment by shweta1234567** [comment]
@keharper Yes I have checked this and this format creates a full shipment.

**Comment by shweta1234567** [comment]
@jeff-matthews I have posted my question Magento Stack Exchange but they didn't reply me can you help me to find the solution.
